bodhi/tests/server/services/test_updates.py

```diff
@@ -291,6 +291,51 @@ def test_new_rpm_update_koji_error(self, publish, *args):
         self.assertEquals(up['errors'][0]['description'],
                           "Koji error getting build: bodhi-2.0.0-2.fc17")
 
+    @mock.patch(**mock_valid_requirements)
+    @mock.patch('bodhi.server.notifications.publish')
+    def test_koji_config_url(self, publish, *args):
+        """
+        Test html rendering of default build link
+        """
+        self.app.app.registry.settings['koji_web_url'] = u'https://koji.fedoraproject.org'
+
+        nvr = u'bodhi-2.0.0-2.fc17'
+        resp = self.app.post_json('/updates/', self.get_update(nvr))
+        resp = self.app.get('/updates/%s' % nvr, headers={'Accept': 'text/html'})
```

Comment:
Let's add a space above this line, since we are testing the response from the GET call here.